src/Handler/CommandNameExtractor/NamedCommandExtractor.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace League\Tactician\Handler\CommandNameExtractor;
```

Comment:
On a personal level, I don't think so. Having everything in a common namespace / folder really brings them together.

We've got to name the folder though, so we often assign it a name given to the most important concept, which being important, is also implemented as an object or interface inside that folder. Cheesy example: If I was doing an email library, I might have `Foo\Email\Sender` and `Foo\Email\Recipient` as value objects, but I'd probably assign them both to `Foo\Email\Email`...cause it's an actual concept too.